File: src/main/java/com/tartner/dancehours/querymodel/danceuser/DanceUserListTO.java
package com.tartner.dancehours.querymodel.danceuser;

import java.util.List;
import java.util.UUID;

public class DanceUserListTO {
    private UUID userId;
    private String email;
    private List<DanceUserListRoleListTO> roles;

    public DanceUserListTO() {}

    public DanceUserListTO( final UUID userId, final String email,
        final List<DanceUserListRoleListTO> roles ) {
        this.userId = userId;
        this.email = email;
        this.roles = roles;
    }

    public UUID getUserId() { return userId; }
    public void setUserId( final UUID userId ) { this.userId = userId; }
    public String getEmail() { return email; }
    public void setEmail( final String email ) { this.email = email; }
    public List<DanceUserListRoleListTO> getRoles() { return roles; }
    public void setRoles( final List<DanceUserListRoleListTO> roles ) { this.roles = roles; }
}

File: src/main/java/com/tartner/dancehours/querymodel/danceuser/DanceUserListTO.kt
package com.tartner.dancehours.querymodel.danceuser

import java.util.UUID

public class DanceUserListTO {
    public var userId: UUID? = null
    public var email: String? = null
    public var roles: List<DanceUserListRoleListTO>? = null

    public constructor() {}

    public constructor(userId: UUID, email: String, roles: List<DanceUserListRoleListTO>) {
        this.userId = userId
        this.email = email
        this.roles = roles
    }
}
